server/project/TypescriptBuilder.scala

```diff
@@ -19,9 +19,11 @@ import scala.sys.process.{Process, ProcessLogger}
   * been updated since 2018 and uses Typescript 2 while current Typescript
   * version is 4.
   *
-  * Our custom plugin invokes `tsc` compiler directly. `tsconfig.json` is setup
-  * to compile code from `app/assets/javascripts` directory and outputs result
-  * to `target/web/typescript/javascripts` folder.
+  * Our custom plugin invokes `tsc` compiler directly through the Webpack
```

Comment:
I considered renaming this file, the task name and some of the variables since it now builds CSS as well as converting Typescript to JS.  I opted against it since compiling TS is still the main objective and the CSS is "extra".  I was also hesitant to rename because this PR is already adding a big change and I didn't want to further complicate it.  

Please let me know if you think this is acceptable or if the names will be confusing.

Reply:
I think leaving keeping this class name is fine since, as you said, the CSS is "extra" and you mention it in the comment.

Reply:
How is the CSS "extra"? Will CiviForm not break if it fails to build CSS?

Reply:
That's a good point, Bion.  What I meant is that Webpack is primarily a JavaScript bundler and most of the logic in the file is still around TypeScript/JS.  But, I'll rename the file so as not to cause any confusion.